scripts/buildtools/commands/verify-package.js

```diff
@@ -17,8 +17,10 @@ const EXPECTED_BUNDLE_SIZES = {
   'okta-sign-in.oie.min.js': 1.3 * MB,
   'okta-sign-in.polyfill.js': 504 * KB,
   'okta-sign-in.polyfill.min.js': 108 * KB,
-  'okta-sign-in.next.js': 1.45 * MB,
-  'okta-sign-in.next.no-polyfill.js': 1.3 * MB,
+  //  Revert this when odyssey 1.x migration is complete and legacy package is removed
+  'okta-sign-in.next.js': 1.7 * MB,
+  //  Revert this when odyssey 1.x migration is complete and legacy package is removed
+  'okta-sign-in.next.no-polyfill.js': 1.5 * MB,
```

Comment:
I checked the package sizes and these sizes are still necessary as of right now. This comment was first written when the expected package sizes were even larger than 1.7 and 1.5 but they've progressively dropped since then. This work still adds a bit to the bundle size, probably because we had to add some things like Adobe's Leonardo and the VRT stuff, though the numbers should change assuming I drop VRT packages (and I will remove these comments regardless)